mungegithub/github/github.go

```diff
@@ -479,9 +479,9 @@ func timeout(sleepTime time.Duration, c chan bool) {
 	c <- true
 }
 
-func (config *Config) doWaitStatus(pr *github.PullRequest, pending bool, c chan error) {
+func (config *Config) doWaitStatus(pr *github.PullRequest, pending bool, requiredContexts []string, c chan error) {
```

Comment:
Could the contexts be added to the config to avoid passing the parameter everywhere?